cinema/serializers.py

```diff
@@ -114,10 +114,10 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
         Ticket.validate_ticket(
-            attrs["row"], 
-            attrs["seat"], 
-            attrs["movie_session"].cinema_hall, 
-            ValidationError
+            attrs["row"],
+            attrs["seat"],
+            attrs["movie_session"].cinema_hall,
+            ValidationError,
         )
```

Comment:
Ensure that the `validate_ticket` method in the `Ticket` model properly handles exceptions and provides meaningful error messages. This will help in debugging and provide better feedback to API consumers.